projects/igniteui-angular/src/lib/time-picker/time-picker.component.spec.ts

```diff
@@ -15,7 +15,7 @@ import { IgxToggleModule } from '../directives/toggle/toggle.directive';
 import { IBaseCancelableBrowserEventArgs } from '../core/utils';
 
 /* eslint-disable @typescript-eslint/no-use-before-define */
-describe('IgxTimePicker', () => {
+xdescribe('IgxTimePicker', () => {
```

Comment:
I think this was leftover by accident?

Reply:
no :) this tests are failing with my changes, that's why I exclude them :) Nikolay said that they refactor the IgxTimePicker tests and will try with refactored tests when they are pushed

Reply:
Hmmm, OK!

Reply:
@kdinev we prefer to fix those tests in the pickers refactoring branch, so when this PR is merged to master, then we will update the pirkcers refactoring branch and will remove the xdescribe

Reply:
@kdinev if we include this failing tests the grid test won't be executed that's why I just exclude them

Reply:
New time-picker tests are merged and enabled and if there are some problems with the build we will fix them